frontend/public/components/utils/resource-log.jsx

```diff
@@ -69,6 +94,11 @@ export class ResourceLog extends SafetyFirst {
       stale: false,
       status: STREAM_LOADING
     };
+    this.requestFullscreen =
+      document.documentElement.requestFullScreen ||
+      document.documentElement.webkitRequestFullScreen ||
+      document.documentElement.mozRequestFullScreen ||
+      document.documentElement.msRequestFullscreen;
```

Comment:
`document.documentElement.msRequestFullscreen` may not be necessary since we don't officially support IE